overview/vaults.md

```diff
@@ -0,0 +1,41 @@
+{% hint style="warning" %} This documentation describes certain core aspects of MakerDAO's protocol that could change over time. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+# Maker Vaults
+
+Maker Vaults are a tool of the Maker Protocol through which users can create (mint) Dai against locked-up collateral.
+
+Users generate Dai by leveraging collateral and repay Dai to destroy their generated balance. This on-chain process ensures complete auditability of circulating Dai and its collateral support.
```

Comment:
I would use mint consistently instead of generate/create

Reply:
Also "destroy generated balance" is not a good term. Use burn here and elsewhere